core/src/main/java/org/elasticsearch/action/get/GetRequest.java

```diff
@@ -101,6 +101,9 @@ public ActionRequestValidationException validate() {
             validationException = ValidateActions.addValidationError("illegal version value [" + version + "] for version type [" + versionType.name() + "]",
                     validationException);
         }
+        if (versionType == VersionType.FORCE) {
```

Comment:
I dunno, the more I think about this, more I think we should not have a hard check on this, with 6.0 it won't break anything on GET requests and I'm not sure the complexity is worth it, it can just be removed in 7.0 entirely.

What do you think?